ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -382,7 +407,8 @@ open class SearchBar: UIView {
         searchTextFieldBackgroundView.addSubview(searchTextField)
         searchTextField.translatesAutoresizingMaskIntoConstraints = false
 
-        constraints.append(searchTextField.leadingAnchor.constraint(equalTo: searchIconImageViewContainerView.trailingAnchor, constant: Constants.searchTextFieldLeadingInset))
+        textFieldLeadingConstraint = searchTextField.leadingAnchor.constraint(equalTo: searchIconImageViewContainerView.trailingAnchor, constant: Constants.searchTextFieldLeadingInset)
+        textFieldLeadingConstraint?.isActive = true
```

Comment:
This doesn't look right. You can still append the constraint to be activated later in line 445 after you save the reference in `textFieldLeadingConstraint`.

Reply:
Use another way to avoid force unwrapper `textFieldLeadingConstraint`